index.js

```diff
@@ -303,6 +303,37 @@ class GithubScm extends Scm {
     }
 
     /**
+     * Get a list of opened Pull Requests for a given repository
+     * @method _getOpenedPrs
+     * @param  {Object}      config
+     * @param  {String}      config.scmUri  The scmUri to get opened PRs from
+     * @param  {String}      config.token   The token used to authenticate with the SCM
+     * @return {Promise}
+     */
+    _getOpenedPrs(config) {
```

Comment:
Good catch! It actually needs to be `PRs`: https://github.com/screwdriver-cd/scm-base/blob/master/index.js#L296